pkg/auth/authenticator/bearertoken/bearertoken.go

```diff
@@ -43,5 +46,18 @@ func (a *Authenticator) AuthenticateRequest(req *http.Request) (user.Info, bool,
 	}
 
 	token := parts[1]
-	return a.auth.AuthenticateToken(token)
```

Comment:
It's not obvious to me how this produced a 403 before.  Is `AuthenticateToken` resulting in a forbidden error?  If so, wouldn't we simply continue returning that?